### ISSUE TITLE
Please document minimum versions of ember-cli and ember-cli-babel.

### ISSUE THREAD

**Issue by Gaurav0** [ef4/ember-auto-import#309]
It seems we are not compatible with ember-cli 2.18 and ember-cli-babel 5.1.7

**Comment by ef4** [comment]
Yes, we should document these, or better yet throw a helpful error when we see them.

But also note this is a weird combination. ember-cli 2.18 shipped with ember-cli-babel `^6.6.0`, so using it with babel 5 was already a less-well-tested combination even back when ember-cli 2.18 was current and supported.

**Comment by brkn** [comment]
What's the _**known**_ minimum compatible version of ember-cli?

**Comment by ef4** [comment]
We test against 2.18.
